### DISCUSSION TITLE
2024/09/19/note-1/

### DISCUSSION THREAD

**Discussion by giscus** [licyk/licyk.github.io#19] [Comment]
# 2024/09/19/note-1/

学艺不精，内心脆弱，也经受不起大的挫折… 现实的压力不断增大，麻烦的事情接踵而来… 学不会的东西还是学不会，即使尝试去努力也改变不了是菜鸡的事实… 走在人生的漫漫长路，困惑，迷茫… 不知道该如何是好…

https://licyk.netlify.app/2024/09/19/note-1/

**Comment by Charlotte-Macbeth** [comment]
没关系的，且走且看，最后迟早会找到路。就算没有路，你也会走出自己的路

**Reply by licyk** [reply]
谢谢你的关心

**Comment by stickland1101** [comment]
up主开心一点，没什么坎过不去的，发现自己并不厉害并不代表自己一无是处。博客里面的合集嘎嘎好，都是宝藏，很用心！
